packages/app-earn-ui/src/components/molecules/TabBar/TabBar.tsx

```
@@ -0,0 +1,69 @@
+        width: activeTab.offsetWidth,
+      })
+    }
+  }, [activeIndex])
+
+  return (
+    <div className={styles.tabBar}>
+      <div className={styles.tabHeaders}>
+        {tabs.map((tab, index) => (
+          <button
+            key={index}
+            ref={(el) => (tabRefs.current[index] = el)}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)